lib/CUDD/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
 set(SOURCES
        cuddAddNeg.c
        cuddBddCorr.c
-       cuddDecomp.c
+       cuddDecomp.cpp
```

Comment:
See the error file below:
[err.txt](https://github.com/unsw-corg/SVF/files/1021181/err.txt).

Please keep *.c extension instead of "*.cpp" for the third-party lib CUDD. Any reason behind for the changes?